src/strategies/get-xkcd/schema.mjs

```diff
@@ -0,0 +1,41 @@
+export const schema = {
```

Comment:
love that you're expecting a schema from the data feed! @il3ven we should do this in all strategies or even make it a requirement to be added to any strategy implementation!

Reply:
Just a thought. We can ask every strategy to have a schema file and then lifecycle manager could validate it automatically and crash on invalid incoming message.

Reply:
agreed, I think that'd be a nice thing to have

Reply:
That would be indeed a good idea. Should we have an issue about it? Also: should I in the meantime remove the validation from the sample or can I leave it there until the more streamlined solution is implemented?

Reply:
IMO u should leave the schema as is and I'll create an issue to generalize

Reply:
issue: https://github.com/neume-network/strategies/issues/231